odnoklassniki-android-sdk/src/main/java/ru/ok/android/sdk/Odnoklassniki.java

```diff
@@ -409,10 +437,41 @@ private void signParameters(final Map<String, String> params) {
         params.put(Shared.PARAM_SIGN, sig);
     }
 
-    public final void setOkListener(OkListener listener) {
+    public final void setOkListener(final OkListener listener) {
```

Comment:
Explicit javadoc is required telling about:
1.  What problem are you solving through the deferred responses
2. The difference of response sending process depending on whatever the listener was set before or not (since it functions differently in such a case now).
Same documentation should also be available in affected methods.